charts/argo-cd/templates/argocd-applicationset/clusterrolebinding.yaml

```
@@ -13,4 +13,12 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "argo-cd.applicationSet.serviceAccountName" . }}
     namespace: {{ include "argo-cd.namespace" . }}
+{{- $namespaces := index .Values.configs.params "applicationsetcontroller.namespaces" -}}
+{{- range $namespace := (split "," $namespaces) }}
+  {{- if $namespaces }}
```

This would be clearer ?
```suggestion
  {{- if $namespace }}
```

---

Great! I made the changes as per your suggestion!